EasyPost/Shipment.cs

```diff
@@ -134,6 +134,29 @@ public void GetRates()
             rates = request.Execute<Shipment>().rates;
         }
 
+        /// <summary>
+        ///     Refresh the rates for this Shipment.
+        /// </summary>
+        /// <param name="parameters">Optional dictionary of parameters for the API request.</param>
+        /// <exception cref="ResourceAlreadyCreated">Shipment has not be created server-side yet.</exception>
+        public void RefreshRates(Dictionary<string, object> parameters = null)
+        {
+            if (id == null)
+            {
+                throw new ResourceNotCreated("Shipment must be created before refreshing rates. Try running `shipment.Create()` first.");
+            }
+
+            Request request = new Request("shipments/{id}/rerate", Method.POST);
+            request.AddUrlSegment("id", id);
+            if (parameters != null)
+            {
+                request.AddBody(parameters);
+            }
+
+            Shipment _shipment = request.Execute<Shipment>();
```

Comment:
This variable is not declared as private, why does it start underscore?

Reply:
Oh, that's just something I do for throwaway variables. C# complains about it though. Removed.